### ISSUE TITLE
Colors with functional notation produce redundant non-parameterized functions

### ISSUE THREAD

**Issue by nidico** [matheus23/elm-tailwind-modules#21]
Similar issue as #18, but caused by another color syntax in tailwind config.

CSS colors can be defined in functional notation, e.g. `hsl(30deg 100% 50%)`, see [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value) for a variety of possible syntax definitions.

When this syntax is used, elm-tailwind-modules 0.5.0-alpha3 generates the appropriate colors in `Tailwind/Theme.elm`, but also generates (redundant) functions in `Tailwind/Utilities.elm`.

In the following example, functions like `bg_by_hsl` still exist in the generated `Tailwind/Utilities.elm`, despite color functions like `by_hsl` being generated in `Tailwind/Theme.elm`:

```js
module.exports = {
  theme: {
    colors: {
      'by-name': 'rebeccapurple',
      'by-hex': '#00ff00',
      'by-rgb': 'rgb(214, 122, 127)',
      'by-hsl': 'hsl(30, 100%, 50%)',
      'by-hsl-deg': 'hsl(30deg 100% 50%)',    
      'by-hsla': 'hsla(30, 100%, 50%, .3)',
      'by-hwb': 'hwb(1.5708rad 20% 10% / 0.7)',
    },
  },
}
```

**Comment by matheus23** [comment]
I've published version 0.5.0-alpha5, which should fix the issue.

**Comment by nidico** [comment]
Fix confirmed.
